carma-messenger-core/cpp_message/test/test_encode_decode.cpp

```diff
@@ -203,8 +207,8 @@ TEST(CppMessageTest, testEncodeControlMsg2)
     params.schedule = schedule;
     // TrafficControlDetails START
     j2735_v2x_msgs::msg::TrafficControlDetail detail;
-    detail.choice = j2735_v2x_msgs::msg::TrafficControlDetail::CLOSED_CHOICE;
-    detail.closed = j2735_v2x_msgs::msg::TrafficControlDetail::OPENLEFT;
+    detail.choice = j2735_v2x_msgs::msg::TrafficControlDetail::MINPLATOONHDWY_CHOICE;
+    detail.minplatoonhdwy = 10;
```

Comment:
I suggest you add a new unit test for those fields instead of changing the existing ones. Also add a test for platoonsize as well

Reply:
I changed this unit test back, and added two new tests using the minplatoonhdwy and maxplatoonsize fields. I tried removing the excess geometry and package fields but it's hard to get it to not fail.